docs/QUICKSTART.md

```diff
@@ -0,0 +1,418 @@
+
+#### ✨✨✨ Congrats you have deployed you first **🌴 Sway** Contract on you own local node. ✨✨✨
+
+## 💚 Create a Fuel DApp to interact with your Contract
+
+Now we are going to;
+
+1. **Initialize a React project.**
+2. **Install the `fuels` SDK dependencies.**
+3. **Modify the App.**
+4. **Run our project.**
+
+#### 4.1 Initialize a React project.
+
+To split better our project let's create a new folder `frontend` and initialize our project inside it.
+
+Inside `my-fuel-dapp` run;
+
+```sh
+npx create-react-app frontend
+```
+
+The command will generate a react app using [`Create React App`](https://create-react-app.dev/).
+
+#### 4.2 Install the `fuels` SDK dependencies.
+
+On this step we need to install 3 dependencies to the project;
+
+1. `fuels`: The umbrella package that includes all the main tools; `Wallet`, `Contracts`, `Providers` and more.
+2. `typechain`: Typechain is the ABI TypeScript generator.
+3. `typechain-target-fuels`: The Typechain Target is the specific interpreter for the [Fuel ABI Spec](https://github.com/FuelLabs/fuel-specs/blob/master/specs/protocol/abi.md).
+
+> ABI stands for Application Binary Interface. ABI's inform the application the interface to interact with the VM, in other words, they provide info to the APP what methods a contract has, what params, types it expects, etc...
+
+##### Install
+
+Inside `my-fuel-dapp/frontend` run;
+
+```sh
+npm install fuels --save
+npm install typechain typechain-target-fuels --save-dev
+```
+
+##### Generating contract types
+
+To make easer to interact with contract we use `typechain` to interpret the output ABI JSON from our contract. This JSON was created on the moment we executed the `forc build` to compile our Sway Contract into binary. If you see the folder `my-fuel-dapp/contract/out` you will be able to see the ABI JSON there. If you want to learn more, read the [ABI Specs here](https://github.com/FuelLabs/fuel-specs/blob/master/specs/protocol/abi.md).
+
+Inside `my-fuel-dapp/frontend` run;
+
+```sh
+npx typechain --target=fuels --out-dir=./src/contracts ../contract/out/debug/*-abi.json
+```
+
+You should see something like this;
+
+```sh
+Successfully generated 4 typings!
+```
+
+Now you should be able to find a new folder `my-fuel-dapp/frontend/src/contracts`. This folder was auto-generated by our `typechain` command, this files abstract the work we would need to do, to create a contract instance, and also generate a complete TypeScript interface to the Contract, making easy to develop.
+
+### 3. Modify the App.
+
+Inside the `frontend` folder let's add a code that interact with our contract.
+Read the comments to help you understand the App parts.
+
+Change the file `my-fuel-dapp/frontend/src/App.tsx` to;
```

Comment:
```suggestion
Change the file `my-fuel-dapp/frontend/src/App.tsx` to:
```
Another thing to note is that while going through the quickstart the create-react-app command did not generate an App.tsx file.  Instead I have an App.js file locally.  This could be confusing for new devs.